docs/rofl/app.md

```diff
@@ -131,17 +131,25 @@ The simplest way to test and debug your ROFL is with a local stack.
     }
    ```
 
-2. Compile ROFL in the _unsafe_ mode:
+2. Navigate to `examples/runtime-sdk/rofl-oracle` and compile
+ROFL in the _unsafe_ mode:
 
    ```shell
    oasis rofl build sgx --mode unsafe
    ```
 
-3. Spin up the Sapphire Localnet docker container and mount your `rofl-oracle`
+3. Make sure to use the latest Sapphire Localnet docker image
```

Comment:
Since this is a minor point could also just move this into the shell before `docker run`,  instead of adding a new step. 
But this is fine as well, use whichever you prefer.

```shell
# Make sure to use the latest Sapphire Localnet docker image
docker pull ghcr.io/oasisprotocol/sapphire-localnet:latest
# Assuming you are running this command from the `rofl-oracle` directory.
docker run -it -p8544-8548:8544-8548 -v .:/rofls ghcr.io/oasisprotocol/sapphire-localnet
```